### DISCUSSION TITLE
401 Unauthorized - Sign in problem

### DISCUSSION THREAD

**Discussion by artinrexhepii** [lichess-org/mobile#612] [Q&A] [answered]
I have cloned this project, and I wanted to connect with online servers of liches by changing the constants to :
const kLichessHost = String.fromEnvironment(
  'LICHESS_HOST',
 // old version defaultValue: 'http://localhost:9663',
 //current version defaultValue: 'https://lichess.org',
);

const kLichessWSHost = String.fromEnvironment(
  'LICHESS_WS_HOST',
//old version  _defaultValue: 'ws://localhost:9664',_
//current version  defaultValue: 'wss://socket.lichess.org',
);

Everything is working, except SIGN IN.  When I am trying to sign in I am, having this message on terminal:
 GET https://lichess.org/api/account responded with status 401 Unauthorized
 
 Any ideas how to fix it?

**Comment by HaonRekcef** [comment, marked as answer]
Someone correct me if I am wrong, but there is no fix and this is due to security measures on the server side. If you want to check features that need an account, either setup a local lila instance or change the constants to https://lichess.dev and wss://socket.lichess.dev.
